cloud/common/deploy/provider/pulumi.go

```diff
@@ -196,6 +198,11 @@ func parsePulumiError(err error) error {
 
 // Up - automatically called by the Nitric CLI via the `up` command
 func (s *PulumiProviderServer) Up(req *deploymentspb.DeploymentUpRequest, stream deploymentspb.Deployment_UpServer) error {
+	// Verify if dependencies are available
+	if err := checkDependencies(checkPulumiAvailable, checkDockerAvailable); err != nil {
```

Comment:
Missing docker buildx?

Reply:
Not sure this is necessary so will leave it out for now